bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/library/types/DateTimeGroupFunction.java

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright (c) 2014,2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.smarthome.core.library.types;
+
+import java.time.ZonedDateTime;
+import java.util.Set;
+
+import org.eclipse.smarthome.core.items.GroupFunction;
+import org.eclipse.smarthome.core.items.Item;
+import org.eclipse.smarthome.core.types.State;
+import org.eclipse.smarthome.core.types.UnDefType;
+
+/**
+ * This interface is a container for dimension based functions that require {@link DateTypeType}s for its calculations.
```

Comment:
`for dimension based functions`
These group functions are not dimension based but specifically designed for the DateTimeItem base item type.

Reply:
Good catch. Too many copy-pastes ;). Fixed.